config/config.go

```diff
@@ -331,8 +342,11 @@ func (c *Cluster) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if c.Scheme != "http" && c.Scheme != "https" {
 		return fmt.Errorf("`cluster.scheme` must be `http` or `https`, got %q instead for %q", c.Scheme, c.Name)
 	}
-	if c.HeartBeatInterval == 0 {
-		c.HeartBeatInterval = Duration(time.Second * 5)
+	if c.HeartBeatInterval != 0 && c.HeartBeat.Interval != defaultHeartBeat.Interval {
+		return fmt.Errorf("cannot be use `heartbeat_interval` with `heartbeat.interval`")
+	}
+	if c.HeartBeat.Interval == 0 && c.HeartBeat.Timeout == 0 && c.HeartBeat.Response == "" {
+		return fmt.Errorf("`cluster.heartbeat` cannot be unset for %q", c.Name)
```

Comment:
Does this mean that all the existing configs will break after upgrading to new version since they have no `HeartBeat` section?

Reply:
All existing configs would be work.
But if you add `HeartBeat` section with `interval` it throws an error because it must be set once.

Reply:
Then it would be great adding the corresponding tests, so this behavior doesn't accidentally break.

Reply:
It's being tested here.
- [old configuration](https://github.com/Vertamedia/chproxy/pull/60/files#diff-bfeae88573cf9afa7638e19a4f7b4535R83-R84)
- [new configuration](https://github.com/Vertamedia/chproxy/pull/60/files#diff-bfeae88573cf9afa7638e19a4f7b4535R127-R142)
- [errors](https://github.com/Vertamedia/chproxy/pull/60/files#diff-bfeae88573cf9afa7638e19a4f7b4535R424-R438)